src/main/java/org/obiba/es/mica/query/RQLQuery.java

```diff
@@ -639,7 +640,10 @@ public Boolean visit(ASTNode node) {
             List<SortBuilder> sortBuilders = Lists.newArrayList();
             if (node.getArgumentsSize() >= 1) {
               for (int i = 0; i < node.getArgumentsSize(); i++) {
-                sortBuilders.add(processArgument(node.getArgument(i).toString()));
+                SortBuilder sortBuilder = processArgument(node.getArgument(i).toString());
+                ((FieldSortBuilder) sortBuilder).unmappedType("string");
+                sortBuilder.missing("_last");
+                sortBuilders.add(sortBuilder);
```

Comment:
Pourquoi il n'y a pas le test ici ? Disparu pendant le déplacement vers le plugin ? ...

Reply:
https://github.com/elastic/elasticsearch/issues/20853, There is an issue with SortBuilder's toString. Should be fixed by 2.4.7 https://github.com/elastic/elasticsearch/pull/26526.

I could add the old code but comment it because we still would like to test this.